src/EnergyPlus/SteamBaseboardRadiator.hh

```diff
@@ -209,15 +220,6 @@ struct SteamBaseboardRadiatorData : BaseGlobalStruct
     int NumSteamBaseboardsDesign = 0;
     int SteamIndex = 0;
 
-    Array1D<Real64> QBBSteamRadSource;    // Need to keep the last value in case we are still iterating
-    Array1D<Real64> QBBSteamRadSrcAvg;    // Need to keep the last value in case we are still iterating
-    Array1D<Real64> ZeroSourceSumHATsurf; // Equal to the SumHATsurf for all the walls in a zone
-                                          // with no source
-
-    // Record keeping variables used to calculate QBBRadSrcAvg locally
-    Array1D<Real64> LastQBBSteamRadSrc; // Need to keep the last value in case we are still iterating
-    Array1D<Real64> LastSysTimeElapsed; // Need to keep the last value in case we are still iterating
-    Array1D<Real64> LastTimeStepSys;    // Need to keep the last value in case we are still iterating
```

Comment:
And here these used to be arrays and now they are scalars?

Reply:
Yes, only a single Q term to deal with because there isn't a separate heat source that is unique for each surface like in low temperature radiant and ventilated slab.